app/types/reactNavigation.ts

```diff
@@ -0,0 +1,11 @@
+import {IOrganization} from '../screens/organizations';
+
+export enum ScreenNames {
```

Comment:
Ideally should be done for all screens and inserted into the project's navigator creation functions.